fe/projects/Seller.Portal/src/shared/components/DynamicForm/fields/StringField.js

```diff
@@ -27,13 +27,27 @@ function StringField(props) {
     registry = getDefaultRegistry(),
     rawErrors,
   } = props;
+
+  let value = formData;
+
+  if(typeof value === "string") {
+    let re = /[!@#$%^&*(){},<>/?|'\"+=`~\[|\]|\\\\]/g;
+
+    if(formData.match(re)) {
+      value = formData.replace(re, "");
+    }
+  }
+
   const { title, format } = schema;
   const { widgets, formContext } = registry;
   const enumOptions = isSelect(schema) && optionsList(schema);
   let defaultWidget = enumOptions ? "select" : "text";
   if (format && hasWidget(schema, format, widgets)) {
     defaultWidget = format;
   }
+  if(schema.type === "number") {
```

Comment:
**suggestion:** please rework the lines 44 - 50 so that it has if else statements